{{ cookiecutter.package_name }}/.coveragerc

```
@@ -0,0 +1,34 @@
+[tool.coverage.run]
```

Are they the same heading in their dedicated file?

---

no: https://coverage.readthedocs.io/en/latest/config.html#sample-file

```suggestion
[run]
```